### ISSUE TITLE
Offset for temp/humidity?

### ISSUE THREAD

**Issue by zshh** [Inversion-NL/com.philio#29]
Any plans to add offset for temperature and humidity for PAT02-1B?
This sensors are not very accurate out of the box so offset possibility (+/-) for the temp/humidity values would be really handy.

**Comment by Inversion-NL** [comment]
No, I'm using a driver from Athom and there isn't any functionalities like this in the driver.

**Comment by zshh** [comment]
Hi,

Thank you for your reply. So only way to do this I guess is to ask Athom team for the possibility which I will do as soon as I figure out which section of their GitHub I should post it under.

Cheers

**Comment by jolny** [comment]
Hi,
I thought this was an interesting feature so I talked to Athom about it They said the driver would not provide it as some devices has it built-in, but they gave me some pointers as to how it can be done on a per-device basis. Might give it a shot, I'll keep you updated.

**Comment by Inversion-NL** [comment]
That would be great!

**Comment by RogerGorissen** [comment]
Jolny, did you succeed in adding offset for humidity and temperature? If so, how did you do it? Regads, Roger

**Comment by jolny** [comment]
Nope, sorry. I noted that it would be possible to do, but nothing more :)

Den fre 3 maj 2019 kl 19:06 skrev RogerGorissen <notifications@github.com>:
